packages/import-bundle/src/index.js

```diff
@@ -26,12 +27,14 @@ export async function importBundle(bundle, options = {}) {
   let CompartmentToUse = Compartment;
   if (
     inescapableTransforms.length ||
-    Object.keys(inescapableGlobalLexicals).length
+    Object.keys(inescapableGlobalLexicals).length ||
+    Object.keys(inescapableGlobalProperties).length
```

Comment:
If this is on a hot path, `hasAnyKeys` might be cheaper:

```js
function hasAnyKeys(object) {
  for (const key in object) {
    if (Object.prototype.hasOwnProperty.call(object, key)) {
      return true;
    }
  }
  return false;
}
```

Reply:
Dunno, I was just echoing what was already there...  I really think it's not a hot path because it only happens for static vats, once per.